src/App.jsx

```diff
@@ -1,62 +1,59 @@
-import React, { useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useState } from 'react';
+import { useSelector } from 'react-redux';
 import Body from './Layouts/body/Body';
 import Footer from './Layouts/footer/Footer';
 import Header from './Layouts/header/Header';
 import ErrorBoundary from './Components/ErrorBoundary';
-import { api } from './API/PostService';
-
 import Modal from './Components/UI/modal/Modal';
 import VideoFormModal from './Components/UI/videoformmodal/VideoFormModal';
 import VideoDetails from './Components/VideoDetails';
 import MyButton from './Components/UI/button/MyButton';
 import useModalState from './hooks/useModalState';
 
 function App() {
-  const [videos, setVideos] = useState([]);
+  const videos = useSelector((state) => state.videos);
```

Comment:
`useSelector`
nice!

Reply:
NIT: usually all selectors are move to separate file (because sometimes the grew up to big ones), so they are much readable and easier to use in components
```
const useVideosSelector = useSelector((state) => state.videos);
```

//cmponent
```
const videos = useVideosSelector();
```